src/OrchardCore.Themes/TheAdmin/Assets/scss/modules/theme/darkmode/_forms.scss

```diff
@@ -0,0 +1,8 @@
+
+.has-filter .btn:not(.show) {
+    border-color: #3C4144 !important;
+}
+
+.has-filter .form-control-feedback, .has-search .form-control-feedback {
```

Comment:
Can this be global for both theme why the value is same?!!

Reply:
The value is currently the same, but also could be changed. So since it a color related, it should be part of the theme files. I was hoping for a variable to use here but did not find one that I can use. I am going to make one more change soon so that the icon color in the same too on the dark theme